config.yml

```diff
@@ -13,6 +13,8 @@ defaults:
     allowedPatterns:
       - '^https://www.mozilla.org/'
       - '^https://s3-us-west-2.amazonaws.com/taskcluster-public-artifacts/'
+      - '^https://public-artifacts.taskcluster.net/'
```

Comment:
So long as we're using regex I think that ensuring `^` and `/` is pretty good. Are the `.` in `^https://www.google.com` going to allow all characters if not escaped? Like if I got `https://wwwogoogle.com` would that match?